watod

```diff
@@ -88,55 +75,26 @@ if [[ $# -gt 0 ]]; then
 	COMPOSE_CMD="${COMPOSE_CMD} $@"
 fi
 
-if [ ! -z $DEVEL ]; then
-    echo "DEVELOPER MODE ACTIVATED: Source files have been mounted within /home/bolty when you enter the container"
-    MODULES_DIR="$MONO_DIR/modules/dev_overrides"
-    export MODULES_DIR_EXP="$MODULES_DIR"
+# Allow for local overrides of any of the below parameters
+if [ -f "$MONO_DIR/watod-config.sh" ]; then
+	source "$MONO_DIR/watod-config.sh"
 fi
 
-# generate .env file from watod_scripts/watod-setup-env.sh
-if [ ! -z $VERBOSE ]; then # if we want to see all verbose coming from setting up env
-    cd $MONO_DIR && bash watod_scripts/watod-setup-env.sh
+# Change docker-compose override according to mode of operation
+MODE_OF_OPERATION=${MODE_OF_OPERATION:-"deploy"}
+if [ $MODE_OF_OPERATION == "deploy" ]; then
+	MODULES_DIR="$MONO_DIR/modules"
+elif [ $MODE_OF_OPERATION == "develop" ]; then
+	MODULES_DIR="$MONO_DIR/modules/dev_overrides"
 else
-    cd $MONO_DIR && bash watod_scripts/watod-setup-env.sh &> /dev/null
+	MODULES_DIR="$MONO_DIR/modules"
 fi
 
-if [ ! -z $PRINT_LOCAL_PORTS ]; then
-    # get list of services to traverse
-    SERVICE_LIST="$(run_compose ps --services | sort)"
-
-    # fixes hostnames for VMs so that by default $HOSTNAME is the first element
-    # in the array, which often is the domain name that can be accessed
-    # externally
-    read -ra HOSTNAME <<< $(hostname -A)
-    echo "========================================================================="
-    echo "Ports exposed by running containers:"
-    echo "========================================================================="
-    LOCAL_FORWARD_ALL=""
-
-    PORTS_STRING=""
-    for SERVICE in $SERVICE_LIST; do
-        ENV_VARS="$(run_compose exec "$SERVICE" env || true)"
-        VNC_PORT=$(echo "$ENV_VARS" | grep ^VNC_PORT= | cut -d '=' -f2)
-        VNC_PORT=${VNC_PORT:0:5} # Strip unncessary characters
-
-        if [ ! -z $VNC_PORT ]; then
-            echo "$SERVICE service exposes a VNC Server at $HOSTNAME:$VNC_PORT"
-            
-            LOCAL_FORWARD="-L ${VNC_PORT}:localhost:${VNC_PORT}"
-            LOCAL_FORWARD_ALL="${LOCAL_FORWARD_ALL} $LOCAL_FORWARD"
-            PORTS_STRING="${PORTS_STRING},${VNC_PORT}:${SERVICE}_VNC"
-            echo "	To forward it locally, run"
-            echo "		ssh$LOCAL_FORWARD $USER@$HOSTNAME"
-            echo "	on your local machine attach to VNC at localhost:${VNC_PORT}"
-            echo
-        fi
-    done
-    
-    echo "========================================================================="
-    echo "To forward all ports locally:"
-    echo "ssh $LOCAL_FORWARD_ALL $USER@$HOSTNAME"
-    echo "========================================================================="
```

Comment:
I removed this because its not used anymore, and vnc should be deprecated eventually